admin/package.json

```
@@ -25,7 +25,8 @@
     "lint": "npm run lint:js && npm run lint:hbs && npm run lint:scss",
     "lint:fix": "npm run lint:js -- --fix && npm run lint:hbs -- --fix && npm run lint:scss -- --fix",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:js": "eslint . --cache --cache-strategy content",
+    "lint:js:uncached": "eslint .",
```

Si on exprimait l'intention plutôt que l'implémentation ?
```suggestion
    "lint:js:modified-ony" : "eslint . --cache --cache-strategy content",
    "lint:js:all": "eslint .",
```

---

En anticipant la PR #3721, il sera nécessaire de changer le nommage des tâches  `lint:js` pour ne pas exécuter deux fois le lint

---

> En anticipant la PR #3721, il sera nécessaire de changer le nommage des tâches lint:js pour ne pas exécuter deux fois le lint
`npm-run-all --parallel lint:*` ne va pas chercher après les premiers deux points. Pas de problème ici donc

---

> Si on exprimait l'intention plutôt que l'implémentation ?

Je suis d'accord avec toi mais dans l'idée, il faudrait que l'option avec cache soit naturelle donc `lint:js:modified-only"` n'est pas souhaitable à mon avis et ça fait que la dénomination `lint:js:all` perd de son sens 🤔 

Est-ce que tu as une autre idée ?

---

Comme ça, ça me va bien, je passe en :rocket: ready to merge